package.json

```diff
@@ -26,6 +26,8 @@
   },
   "dependencies": {
     "chalk": "^1.1.3",
-    "minimist": "^1.2.0"
+    "htmltojsx": "git://github.com/harrison/react-magic.git",
```

Comment:
Also, I don't see where any of these packages are being imported and used. Maybe I'm missing something.